### PR TITLE
[AAE-10777] Move in common service the real common services

### PR DESCRIPTION
**Pull request by eromano**
remove Pagination js-api dependencies
move search-input component in content

**Please check if the PR fulfills these requirements**

> - [X] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [X] Tests for the changes have been added (for bug fixes / features)
> - [X] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [X] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

Move in common service the real common services
remove Pagination js-api dependencies
move search-input component in content

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by mauriziovitale — APPROVED** [review]

File: lib/content-services/src/lib/search-text/search-text-input.component.spec.ts [hunk 1]
```diff
@@ -16,14 +16,14 @@
  */
 
 import { ComponentFixture, TestBed, discardPeriodicTasks, fakeAsync, tick } from '@angular/core/testing';
-import { CoreTestingModule } from '../testing/core.testing.module';
+import { CoreTestingModule } from '../../../../core/src/lib/testing/core.testing.module';
```
Comment (DenysVuika) [inline lib/content-services/src/lib/search-text/search-text-input.component.spec.ts:19]: probably wrong path

File: lib/content-services/src/lib/search-text/search-text-input.component.spec.ts [hunk 2]
```diff
@@ -16,14 +16,14 @@
...
 import { SearchTextInputComponent } from './search-text-input.component';
 import { DebugElement } from '@angular/core';
 import { By } from '@angular/platform-browser';
 import { Subject } from 'rxjs';
 import { TranslateModule } from '@ngx-translate/core';
-import { UserPreferencesService } from '../common/services/user-preferences.service';
-import { setupTestBed } from '../testing/setup-test-bed';
+import { UserPreferencesService } from '../../../../core/src/lib/common/services/user-preferences.service';
```
Comment (DenysVuika) [inline lib/content-services/src/lib/search-text/search-text-input.component.spec.ts:25]: wrong paths

File: lib/core/src/lib/services/content.service.ts [hunk 3]
```diff
@@ -19,15 +19,21 @@ import { Injectable } from '@angular/core';
...
 import { AuthenticationService } from '../auth/services/authentication.service';
 import { LogService } from '../common/services/log.service';
 import { catchError } from 'rxjs/operators';
 import { PermissionsEnum } from '../models/permissions.enum';
 import { AllowableOperationsEnum } from '../models/allowable-operations.enum';
 import { DownloadService } from './download.service';
-import { ThumbnailService } from './thumbnail.service';
+import { ThumbnailService } from '../common/services/thumbnail.service';
+
+export interface FolderCreatedEvent {
```
Comment (DenysVuika) [inline lib/core/src/lib/services/content.service.ts:31]: this should probably belong to content lib?

**Review by DenysVuika — CHANGES_REQUESTED** [review]
looks good, only few incorrect paths for import

File: lib/core/src/lib/services/content.service.ts (hunk 3 above)
Comment (eromano) [reply lib/core/src/lib/services/content.service.ts:31]: Thumbnail looks good everywhere IMO

**Pull request merged**